code/tests/functional/backend_api/conftest.py

```diff
@@ -55,10 +55,15 @@ def setup_default_mocking(httpserver: HTTPServer, app_config: AppConfig):
         }
     )
 
+    httpserver.expect_oneshot_request(
```

Comment:
I think it'd be be good to add a comment here about why we're using the onsehot request